Python/Product/Analysis/ModulePath.cs

```diff
@@ -23,10 +23,9 @@
 using System.Text.RegularExpressions;
 using Microsoft.PythonTools.Analysis.Infrastructure;
 using Microsoft.PythonTools.Interpreter;
-using Microsoft.PythonTools.Parsing;
 
 namespace Microsoft.PythonTools.Analysis {
-    struct ModulePath {
+    internal struct ModulePath {
```

Comment:
It is not accessible from VS language server assembly otherwise. `LoadFileAsync` calls it to determine if file is Python. However we could limit it to `.py` and then won't need `ModulePath`

Reply:
Does .NET Core not use `internal` by default for structs?